src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -106,6 +106,22 @@ function cv_m(q_t=0, q_l=0, q_i=0)
 
 end
 
+
+"""
+    moist_gas_constants([q_t=0, q_l=0, q_i=0])
+
+Wrapper to return R_m, cv_m, cp_m, and gamma_m all at once
+"""
+function moist_gas_constants(q_t=0, q_l=0, q_i=0)
+
+    Rm  = gas_constant_air(q_t, q_l, q_i)
```

Comment:
Can we use `_R_m`, `_cv_m`, `_cp_m` for consistent naming of local variables here. And I suggest unicode `\gamma_m` for readability, here and throughout code.